### ISSUE TITLE
Dragging between multiple trees

### ISSUE THREAD

**Issue by noahrubin** [angular-ui-tree/angular-ui-tree#557]
I'm attempting to create an application that contains two trees, where one functions as a board and the other as a drawer of "magnets."  

When I attempt to drag a ui-tree-node (magnet) from the drawer tree to the other, I get the error <b>"TypeError: Cannot read property '$id' of null."</b>  This occurs at <b><em>line 1207</em></b> of the code, at the check if the node was dropped in the same place: 
```javascript
if (this.index === this.sourceInfo.index && this.parent.$nodeScope.$id === this.source.$nodeScope.$id) {
     console.log("DO NOTHING, NO CHANGE");
     return;
 }
```
I fixed the code for my purposes by doing this:
```javascript
if (((this.index === this.sourceInfo.index) && ((this.parent.$nodeScope && this.source.$nodeScope) && (this.parent.$nodeScope.$id === this.source.$nodeScope.$id))) ||
((this.source.$parentNodesScope && this.parent) && (this.source.$parentNodesScope.$id === this.parent.$id))) {
     console.log("DO NOTHING, NO CHANGE");
     return;
}
```
However, I am afraid this is not a long term solution, and was wondering if you could look into this use case.  I am currently using v2.6.0, and v1.4.3 of Angular.  Any help is much appreciated!

**Comment by StefanCodes** [comment]
I believe this was introduced in v2.6.0 and fixed in v2.7.0: https://github.com/angular-ui-tree/angular-ui-tree/pull/512

**Comment by noahrubin** [comment]
My bad, thank you @StefanCodes
